src/pages/ManageWallet/TransactionBox.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Transaction } from 'store/ducks/transaction/actionTransactions';
+import Spinner from 'components/Spinner';
+import { ReactComponent as CloseIcon } from 'assets/images/close.svg';
+import { transactionHashToLink } from 'utils/transaction';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import Link from 'components/Link';
+import { SupportedNetworkName } from 'constants/network';
```

Comment:
^ minor, and probably need a lint rule for this, but could you put library imports on top and space out other ones (possibly group those that are somewhat similar)